### ISSUE TITLE
Bugs in the code network and computes for Apply stack implementation

### ISSUE THREAD

**Issue by rberger** [sparkleformation/sparkle-guides#7]
In the _Apply stack implementation_ section of the Stack guild the initial ./sparkleformation/network.rb file has bugs. All the `cider_block join!` statements had a parenthesis in the wrong place.

With Bug in the `dynamic!(ec2_vpc, :network)` and `dynamic!(ec2_subnet, network)` blocks

```
cidr_block join!(ref!(:cidr_prefix, '.0.0/24'))
```

Without bug (Parenthesis should be after `cider_prefix`)

```
cidr_block join!(ref!(:cidr_prefix), '.0.0/24')
```

And in the `outputs` block:

```
network_cidr.value join!(ref!(:cidr_prefix), '.0.0/24')
```

**Comment by rberger** [comment]
In `computes.rb` in both `dynamic!(:ec2_instance, ...)` blocks there was `image_type` and it should be `instance_type`.

The `instance_type` for the `small` instance was also a `t2.micro`. I presume it should be a `t2.small`

Should probably make the ami a parameter since its region specific, etc

**Comment by webframp** [comment]
Saw this same issue with the example template for the _apply and nested_ walkthrough

**Comment by chrisroberts** [comment]
Fixed in #8. Thanks @rberger !
